src/main/java/io/fabric8/forge/generator/github/GithubImportPickRepositoriesStep.java

```diff
@@ -88,6 +94,21 @@ public void validate(UIValidationContext context) {
         if (!value.iterator().hasNext()) {
             context.addValidationError(gitRepositoryPattern, "You must select a repository to import");
         }
+        // Check for repos with already existing bc
+        Controller controller = new Controller(kubernetesClient);
+        OpenShiftClient openShiftClient = controller.getOpenShiftClientOrNull();
+        if (openShiftClient == null) {
+            context.addValidationError(gitRepositoryPattern, "Could not create OpenShiftClient. Maybe the Kubernetes server version is older than 1.7?");
+        }
+        Iterator<GitRepositoryDTO> it = value.iterator();
+        while (it.hasNext()) {
+            GitRepositoryDTO repo = it.next();
+            BuildConfig oldBC = openShiftClient.buildConfigs().withName(repo.getName()).get();
```

Comment:
this should have a `.inNamespace()` call as otherwise it will use the `current namespace` which is slightly random. So it'd be nicer to use:
```java
 openShiftClient.buildConfigs().inNamespace(usersNamespace).withName(repo.getName()).get();
```

the trickier bit is figuring out reliably the current users namespace (where the BC's get created)